test/support/user_controller_annotated.ex

```diff
@@ -27,6 +27,7 @@ defmodule OpenApiSpexTest.UserControllerAnnotated do
          },
          unauthorized: Unauthorized.response(),
          not_found: NotFound.response(),
+         unprocessable_entity: %Reference{"$ref": "#/components/responses/unprocessable_entity"},
```

Comment:
Is the referenced response defined? In the tests we should show how such references are mean to be used.

Reply:
Like this? https://github.com/open-api-spex/open_api_spex/pull/532/commits/0ccecab534ea1ed425a3bc390b73d761b35a89c7